package.json

```diff
@@ -36,6 +36,7 @@
     "zip-webpack-plugin": "^2.0.0"
   },
   "dependencies": {
+    "@tinymce/tinymce-react": "^3.8.0",
```

Comment:
Let's please use sdk WYSIWYG instead of this. If not sure how to implement, connect with me @gobelius

Reply:
removed, using sdk wysiwyg